docs/introduction_tutorial/intro_tutorial_part2.ipynb

```diff
@@ -0,0 +1,1228 @@
+{
```

Comment:
<p>We should suppress this warning. See #300.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>